src/exec/command.rs

```diff
@@ -96,7 +96,7 @@ pub fn execute_commands<I: Iterator<Item = io::Result<Command>>>(
 pub fn handle_cmd_error(cmd: Option<&Command>, err: io::Error) -> ExitCode {
     match (cmd, err) {
         (Some(cmd), err) if err.kind() == io::ErrorKind::NotFound => {
-            print_error(format!("Command not found: {:?}", cmd));
+            print_error(format!("Command not found: {:?}", cmd.get_program()));
```

Comment:
Minor: by using `{:?}`, we further rely on Debug outputs which are typically not guaranteed to be stable. Furthermore, `.get_program()` returns an `OsStr` and I'm not sure what it would do in case of invalid UTF-8 program names (which, agreed, is hopefully a non-existent corner case). But still, taking these two things together, I think I would prefer something more explicit like (untested):
```suggestion
            print_error(format!("Command not found: '{}'", cmd.get_program().to_string_lossy()));
```

Reply:
Sure! That is a good idea 🙂  Did in 9c9097f